Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2153,6 +2155,13 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C17I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
     },
+    '2026D97' : {
+        'Geom' : 'Extended2026D97',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T25',
+        'Era' : 'Phase2C17I13M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
```

Comment:
as this is waiting for signatures, do you mind adding the step `ALCAPhase2` here, as done for the other geometries in https://github.com/cms-sw/cmssw/pull/39858 ?

Reply:
Done.